core/src/main/java/com/googlecode/psiprobe/model/TomcatTestReport.java

```diff
@@ -27,8 +27,8 @@
   /** The Constant TEST_FAILED. */
   public static final int TEST_FAILED = 2;
 
-  /** The default memory size. */
-  private final int defaultMemorySize = 1024 * 1024; // 1MB
+  /** The default memory size. 1 MB */
```

Comment:
Oh, looks like the field needs to be `static` too. Of course, it will show up as a 7-digit number...